protocol-designer/src/steplist/formLevel/handleFormChange/dependentFieldsUpdateMix.ts

```diff
@@ -56,39 +59,21 @@ const updatePatchOnPipetteChannelChange = (
 ): FormPatch => {
   if (patch.pipette === undefined) return patch
   let update = {}
-  const prevChannels = getChannels(rawForm.pipette as string, pipetteEntities)
-  const nChannels =
+  const previousChannels = getChannels(
+    rawForm.pipette as string,
+    pipetteEntities
+  )
+  const nextChannels =
     typeof patch.pipette === 'string'
-      ? getChannels(patch.pipette, pipetteEntities)
+      ? getChannels(patch.pipette as string, pipetteEntities)
```

Comment:
```suggestion
      ? getChannels(patch.pipette, pipetteEntities)
```

not needed since you checked type already